pallets/order-book/src/lib.rs

```diff
@@ -1020,7 +1016,7 @@ pub mod pallet {
 		fn get_order_details(order: Self::OrderId) -> Option<Swap<T::Balance, T::AssetCurrencyId>> {
 			Orders::<T>::get(order)
 				.map(|order| Swap {
-					amount: order.buy_amount,
+					amount: todo!(),
```

Comment:
And here too

Reply:
Right now, FI only checks the asset ids and does not require the amount. I suppose in an earlier version of FI, the amount was required. I propose to remove the amount entirely from the return type and only report back the `asset_out._d` and `asset_in_id`. If we want to keep it, it should be `amount_out`.

Reply:
Thanks for the details on this!

Reply:
Given that `Swap` always expects the `amount` as `currenc_in`, I think it can be interesting to let this with `order.amount_in` value, which means the current fulfilled amount of this order. And makes sense for an API call called `get_order_details()`.  WDYT?

Reply:
Sounds good to me!